backend/src/BuiltinCliHost/Libs/Cli.fs

```diff
@@ -19,6 +19,11 @@ module WT = LibParser.WrittenTypes
 module Json = BuiltinExecution.Libs.Json
 
 
+let typ =
+  FQTypeName.Package
+    { owner = "Darklang"; modules = [ "Cli" ]; name = "ExecutionError"; version = 0 }
```

Comment:
love that this is extracted, but let's rename to `executionErrorTyp` or something like that

Reply:
or maybe something like

```fsharp
module ExecutionError = 
  let fqTypeName = FQTypeName....
  let typeRef = ...
```

since many/most uses are type refs